proto/controlclient.proto

```diff
@@ -83,3 +84,16 @@ message _FlushCacheRequest {
 message _FlushCacheResponse {
 
 }
+
+message _GenerateTokenRequest {
+  string session_token = 1;
```

Comment:
should we keep the session token here for now, or add it in later?

Reply:
It's passed as part of the authentication headers, so it doesn't need to be part of the protos. We'll need to add some special handling in the momento auth context to be able to say "only session tokens can do this action" (our first authorization checks!) but that will be handled within control plane.